src/blackbird.py

```diff
@@ -51,14 +54,18 @@ def LearnFromExamples(self, examples):
 
     # Overriden from MCTS
     def SampleValue(self, state, player):
-        raise NotImplementedError
+        value = self.getEvaluation([state.NumericRepresentation()])
+        return value, player
 
     def GetPriors(self, state):
-        raise NotImplementedError
+        return self.getPolicy([state.NumericRepresentation()])
 
     # Need to be overriden
     def NewGame(self):
-        raise NotImplementedError
+        return BoardState()
 
```

Comment:
I actually deprecated the need for this function (if there ever was one?). We can just take it in favor of BoardState()

Reply:
<a href="https://github.com/jordan-singer/BlackBird/commit/e178e84a1d2fe929f73c6bd46fa7b64105a2f3b7">Resolved</a>.